yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/extension/ThirdPartyAttributeContent.java

```diff
@@ -0,0 +1,32 @@
+package com.yoti.api.client.shareurl.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.yoti.api.client.AttributeDefinition;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+public class ThirdPartyAttributeContent {
+
+    private final Date expiryDate;
+
+    @JsonProperty("definitions")
```

Comment:
Should we put this annotation on the public property, not the private one?

Reply:
Don't think so - generally you put the annotation on the field declaration, unless you need it to be different (i.e. in the case of the expiry date I want it to be a string of a certain format)

Reply:
I think the JSON serializer in .NET (newtonsoft) takes all of the public properties/fields of a class and serializes them. So this would get added to the payload twice. But as long as it doesn't do that in Java, then this is fine!